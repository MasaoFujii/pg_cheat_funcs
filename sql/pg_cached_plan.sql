CREATE EXTENSION pg_cheat_funcs;

\pset null '(null)'

PREPARE stmt AS SELECT * FROM pg_database WHERE datname = $1;
SELECT * FROM pg_cached_plan_source('stmt');

DROP EXTENSION pg_cheat_funcs;
