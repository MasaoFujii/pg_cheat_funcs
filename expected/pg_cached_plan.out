CREATE EXTENSION pg_cheat_funcs;
\pset null '(null)'
PREPARE stmt AS SELECT * FROM pg_database WHERE datname = $1;
SELECT * FROM pg_cached_plan_source('stmt');
 generic_cost | total_custom_cost | num_custom_plans | force_generic | force_custom 
--------------+-------------------+------------------+---------------+--------------
           -1 |                 0 |                0 | f             | f
(1 row)

DROP EXTENSION pg_cheat_funcs;
